libtransmission/transmission.h

```diff
@@ -613,11 +614,11 @@ bool tr_sessionGetAntiBruteForceEnabled(tr_session const*);
 void tr_torrentStartNow(tr_torrent*);
 
 /** @brief Return the queued torrent's position in the queue it's in. [0...n) */
-int tr_torrentGetQueuePosition(tr_torrent const*);
+ssize_t tr_torrentGetQueuePosition(tr_torrent const*);
```

Comment:
If the range is `[0...n)` shoudn't this be `size_t`?

Also, the `ssize_t` has portability issues and should be avoided. If we _really_ need signed values here, make this an `int32_t`

Same comments apply to the other queue functions below

Reply:
OK, done.